src/Ideas.js

```diff
@@ -8,91 +8,116 @@ import './Ideas.css';
 
 slug.defaults.mode = 'rfc3986';
 
-const defaultSelection = ['Economic Development', 'Data and Expertise', 'Resiliency and Sustainability', 'Neighborhood Improvement', 'Housing'];
+const allCategories = ['Economic Development', 'Data and Expertise', 'Resiliency and Sustainability', 'Neighborhood Improvement', 'Housing'];
 
-function checkIfAllSelected(categories, all) {
+const allValues = {};
+
+const checkIfAllSelected = (categories, all) => {
   let allSelected = true;
   all.forEach((d) => {
     if (categories.indexOf(d) < 0) allSelected = false;
   });
   return allSelected;
-}
+};
 
 // utility helpers for arrays
-function removeItem(array, value) {
+const removeItem = (array, value) => {
```

Comment:
Question: why does this need property initializer syntax here? I think the main benefit of it was to bind `this` of the outer scope (or respective class).